### PR TITLE
Always make enum values a string

### PR DESCRIPTION
**Pull request by tversteeg**
This way it's possible to use a strictly typed parser for both the JSON & Lua files. When the number value is needed `calc_value` can still be used as a number.

### CODE REVIEW

File: generator/cpp2ffi.lua [hunk 1]
```diff
@@ -1333,7 +1333,7 @@ function M.Parser()
 				if not prevvalue then
 					value = 0
```
Comment (sonoro1234) [inline generator/cpp2ffi.lua:1334]: I guess this should be
value = "0"

**Comment by oprypin** [comment]
I definitely agree with this change. I'd be able to remove this workaround:
https://github.com/oprypin/crystal-imgui/blob/1a0cc4237191dae7eaeb36ed88ec7563873a7492/generate.cr#L741-L744

File: generator/cpp2ffi.lua (hunk 1 above)
Comment (tversteeg) [reply generator/cpp2ffi.lua]: Thanks, amended this change.

**Comment by sonoro1234** [comment]
Thanks!!

**Pull request merged**